app/javascript/src/components/Dashboard/Notes/NoteCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
+// @ts-ignore
+import { Typography, Avatar, Tag, Dropdown } from "neetoui";
+
+const NoteCard = ({
+  title,
+  body,
+  createdAt,
+  tags = [{ tag: "Getting Started" }],
+}) => (
+  <div className="neeto-ui-border-gray-300 my-4 flex flex-col border-2 px-1">
+    <div className="p-2">
+      <div className="mt-2 flex justify-between px-2">
+        <Typography className="" style="h3">
+          {title}
+        </Typography>
+        <Dropdown icon={MenuVertical} buttonStyle="text" position="auto-end">
+          <li className="my-1">
+            <span className="w-10/12 p-1">Edit</span>
+          </li>
+          <li className="my-1">
+            <span className="w-10/12 p-1">Delete</span>
+          </li>
+        </Dropdown>
+        {/* <Button icon={MenuVertical} style="text" /> */}
```

Comment:
What's the point of keeping a comment and pushing the code or pertinently merge that code to master branch?

Comment coded should be removed before you even make a commit.

Self-Review is a must process in BigBinary. Whatever you are committing, you are accountable for that code. So make sure the code is clean and working and of course all Rails test cases are passing, before asking the reviewer to review.

Moving forward please keep this in mind.

Reply:
I must've missed it. Would keep it in mind further on.